### PR TITLE
[tbb] fix shared tbb library configuration build under *nix

### PR DESCRIPTION
**Pull request by xandox**
**Describe the pull request**

- What does your PR fix? 
fix tbb cmake config for *nix systems if tbb was built not as static library

- Which triplets are supported/not supported? 
same as previous 
- Have you updated the CI baseline? 
not

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
I hope

### CODE REVIEW

File: ports/tbb/portfile.cmake [hunk 1]
```diff
@@ -24,7 +24,17 @@ if (NOT VCPKG_TARGET_IS_WINDOWS)
...
+        set(TBB_LIB_EXT a)
+    else()
+        if (VCPKG_TARGET_IS_LINUX)
+            set(TBB_LIB_EXT "so.2")
+        elseif(VCPKG_TARGET_IS_OSX)
+            set(TBB_LIB_EXT "dylib")
+        else()
+            set(TBB_LIB_EXT "so")
+        endif()
+    endif()
```
Comment (JackBoosY) [inline ports/tbb/portfile.cmake:37]: Does it works? `TBB_LIB_EXT` is only used in `vcpkg_install_msbuild` which can't be used in *inx.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/tbb/portfile.cmake (hunk 1 above)
Comment (xandox) [reply ports/tbb/portfile.cmake:37]: >Does it work?

Yes :)
>TBB_LIB_EXT is only used in vcpkg_install_msbuild which can't be used in *inx.

Not, It also used in ${SOURCE_PATH}/cmake/templates/TBBConfigInternal.cmake.in during file configuration (Line - 105)

**Comment by JackBoosY** [comment]
I will test the usage manually later.

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
Did you pushed the wrong branch?

**Comment by xandox** [comment]
Hell!
Yes. Sorry. I will recreate PR.

**Comment by xandox** [comment]
Created new https://github.com/microsoft/vcpkg/pull/14555
Sorry again. My git-fu not perfect.

**Pull request closed without merging**